spendsterIOS/Presentation/AddressScreen/AddressScreenViewController.swift

```diff
@@ -8,19 +8,30 @@
 
 import UIKit
 
-class AddressScreenViewController: UIViewController {
-
-    
-    
+class AddressScreenViewController: UIViewController, AuthView {
+    var presenter: AddressScreenPresenter?
     @IBOutlet weak var nameTextField: UITextField!
-    @IBOutlet weak var addressTextField: UITextField!
-    @IBAction func continueButtonPressed(_ sender: Any) {
-        // TODO 45-activity-home-screen
+    @IBOutlet weak var phoneNumberTextField: UITextField!
+    @IBOutlet weak var errorMessage: UILabel!
+    @IBAction func getStartedButtonPressed(_ sender: Any) {
+        self.errorMessage.isHidden = true
+        self.presenter?.makeUpdate(name: nameTextField.text!, phoneNumber: phoneNumberTextField.text!)
     }
+
     override func viewDidLoad() {
         super.viewDidLoad()
-        
+        self.errorMessage.isHidden = true
```

Comment:
@Thongvor extract lines 23 - 25 into a func. For example, you can name it like initDefaultUI()